alcs-api/src/comment/comment.controller.ts

```diff
@@ -0,0 +1,100 @@
+import { Mapper } from '@automapper/core';
+import { InjectMapper } from '@automapper/nestjs';
+import {
+  Body,
+  Controller,
+  Delete,
+  ForbiddenException,
+  Get,
+  NotFoundException,
+  Param,
+  Patch,
+  Post,
+  Req,
+  UseGuards,
+} from '@nestjs/common';
+import { ApiOAuth2 } from '@nestjs/swagger';
+import * as config from 'config';
+import { RoleGuard } from '../common/authorization/role.guard';
+import { UserRoles } from '../common/authorization/roles.decorator';
+import { ANY_AUTH_ROLE } from '../common/enum';
+import { CommentDto, CreateCommentDto, UpdateCommentDto } from './comment.dto';
+import { Comment } from './comment.entity';
+import { CommentService } from './comment.service';
+
+@Controller('comment')
+@ApiOAuth2(config.get<string[]>('KEYCLOAK.SCOPES'))
+@UseGuards(RoleGuard)
+export class CommentController {
+  constructor(
+    private commentService: CommentService,
```

Comment:
@Inject(CommentService) private commentService: CommentService

Reply:
Code runs fine without inject, why do I need to add it?